tools/image_processing/imagej2/imagej2_noise/imagej2_noise.py

```diff
@@ -0,0 +1,100 @@
+# Galaxy wrapper for use with Imagej2 by Greg Von Kuster
+"""
```

Comment:
@bgruening Just saw your note letting me decide for this one.  How about this approach - I'll have several more tools coming that follow this model and are probably about as complex as this one.  For the next tool, I'll take the Cheetah approach, and see how that goes.  If all works well, I'll retrofit this tool to use Cheetah as well. Will that work, or would you rather not have this tool committed in its current format and then have it (possibly) significantly changed later.